v8_test.go

```diff
@@ -1541,3 +1542,23 @@ func TestPanicHandling(t *testing.T) {
 	_ = NewIsolate()
 	_ = *f
 }
+
+func TestNewIsolateWithConstraints(t *testing.T) {
+	// Creates a v8 runtime where the memory is limited to 10MB and memory is
```

Comment:
no need to set it as high as 10MB -- 1MB is fine.  Tests should be as lightweight as possible.

Reply:
With 1MB it seems `NewIsolate()` doesn't even terminate. I think there's a minimum required memory to get v8 started. This should probably be also tested as part of the configuration input, i.e. illegal parameters.